src/include/types.h

```diff
@@ -337,15 +337,46 @@ inline ostream& operator<<(ostream& out, const prettybyte_t& b)
   return out << b.v << " bytes";
 }
 
+namespace {
+  inline ostream& format_si(ostream& out, const uint64_t v, const uint64_t n,
+      const int index, const uint64_t mult)
+  {
+    char buffer[32];
+    char u = " KMGTPE"[index];
```

Comment:
Is the leading space intentional? (This may deserve a comment.)

Reply:
The namespace opens a new block so I indented everything in it. Not correct?

Reply:
Sorry, I should have been specific-- I meant the one in the string in line 343. Looking at it again, I see that you're using the index to select from the array, so it almost certainly is.